### DISCUSSION TITLE
lotus-miner storage-deals import-data  datastore: key not found

### DISCUSSION THREAD

**Discussion by kimtony** [filecoin-project/lotus#8513] [Storage Provider] [answered]
### Checklist

- [X] This is **not** a security-related bug/issue. If it is, please follow please follow the [security policy](https://github.com/filecoin-project/lotus/security/policy).
- [X] This is **not** a question or a support request. If you have any lotus related questions, please ask in the [lotus forum](https://github.com/filecoin-project/lotus/discussions).
- [X] This is **not** a new feature request. If it is, please file a [feature request](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Ffeature&template=feature_request.yml) instead.
- [X] This is **not** an enhancement request. If it is, please file a [improvement suggestion](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Fenhancement&template=enhancement.yml) instead.
- [X] I **have** searched on the [issue tracker](https://github.com/filecoin-project/lotus/issues) and the [lotus forum](https://github.com/filecoin-project/lotus/discussions), and there is no existing related issue or discussion.
- [X] I am running the [`Latest release`](https://github.com/filecoin-project/lotus/releases), or the most recent RC(release canadiate) for the upcoming release or the dev branch(master), or have an issue updating to any of these.
- [X] I did not make any code changes to lotus.

### Lotus component

- [ ] lotus daemon - chain sync
- [ ] lotus miner - mining and block production
- [ ] lotus miner/worker - sealing
- [ ] lotus miner - proving(WindowPoSt)
- [ ] lotus miner/market - storage deal
- [ ] lotus miner/market - retrieval deal
- [ ] lotus miner/market - data transfer
- [ ] lotus client
- [ ] lotus JSON-RPC API
- [ ] lotus message management (mpool)
- [ ] Other

### Lotus Version

```text
1.15.0+mainnet+git.0ac1bbc7a+api1.5.0
lotus version 1.15.0+mainnet+git.0ac1bbc7a.dirty20220311
```

### Describe the Bug

i use offfline deal,then

lotus-miner storage-deals import-data bafyreigcoswhvd22ssincex4fol5elufnqt47kgmpzvixxgzgxe3ohb6re /mnt/md0/dea                                               l_data/reducing-the-number-of-high-fatality-accidents.car

### Logging Information

```text
ERROR: failed getting deal bafyreigcoswhvd22ssincex4fol5elufnqt47kgmpzvixxgzgxe3ohb6re: No state for /bafyreigcoswhvd22ssincex4fol5elufnqt47kgmpzvixx                                               gzgxe3ohb6re: datastore: key not found
```

### Repo Steps

1. Run '...'
2. Do '...'
3. See error '...'
...
''

**Comment by Reiers** [comment]
Hi @kimtony 

Thanks for the report. The issue you are experiencing lies elsewhere.

Moving this issue to Lotus Discussions for help and troubleshooting.

Thanks!

Closing Lotus issue ticket.

**Comment by LexLuthr** [comment, marked as answer]
@kimtony This happens when deal proposal never reached the lotus-miner.

https://filecoinproject.slack.com/archives/CPFTWMY7N/p1650539129181209

Let me know if you need any further help on this.

**Reply by kimtony** [reply]
thanks,
My lotus-miner can be reachability, but I still can't import the deal data

lotus-miner net reachability
AutoNAT status:  Public

**Reply by kimtony** [reply]
sorry for my question, lotus is not reachable

After I opened , it works now  

thanks

**Reply by LexLuthr** [reply]
That's good. Let me know if you need any further help on this or are we good to close this discussion?

**Reply by kimtony** [reply]
Thanks, this discussion can be closed
